### DISCUSSION TITLE
OIDC support

### DISCUSSION THREAD

**Discussion by realkarmakun** [pelican-dev/panel#191] [Ideas]
Very much needed. Moreover much more federation log ins can be supported through that (besides Steam and Discord as mentioned in Discord FAQ).

**Comment by realkarmakun** [comment]
Sorry did not see other requests like #173 #121 but I think saying it's OIDC instead of OAuth for specific service is more convenient

**Comment by lancepioch** [comment]
Oauth support is planned, thanks!
